rosalind_cons.py

```diff
@@ -4,10 +4,19 @@ def main(data):
     lines = [text.replace("\n", "") for text in data.split(">")]
     columns = zip(*[text[13:] for text in lines if text != lines[0]])
     consensus = "".join([Counter(column).most_common(1)[0][0] for column in columns])
+    data = {'A': '', 'C': '', 'G': '', 'T': ''}
 
-    a = " ".join([str(Counter(column)['A']) for column in columns])
-    c = " ".join([str(Counter(column)['C']) for column in columns])
-    g = " ".join([str(Counter(column)['G']) for column in columns])
-    t = " ".join([str(Counter(column)['T']) for column in columns])
+    for i, column in enumerate(columns):
 
-    return '%s\nA: %s\nC: %s\nG: %s\nT: %s' % (consensus, a, c, g, t)
+        if i != 0:
+            data['A'] += ' '
+            data['C'] += ' '
+            data['G'] += ' '
+            data['T'] += ' '
```

Comment:
There is a whitespace issue if I don't do this. I'm thinking of a better way to do it.